samples/secure-sql-chatbot/README.md

```diff
@@ -0,0 +1,100 @@
+# Secure chatbot using advanced RAG and a SQL database
+
+![Movie Muse](moviemuse.png)
+
+This example demonstrates how to create a secure chatbot with RAG using
+`quarkus-langchain4j`. This chatbot internally uses LLM-generated SQL
+queries to retrieve the relevant information from a PostgreSQL database.
+
+### Setup
+
+The demo requires that your Google account's full name and email are configured.
+You can use system or env properties, see `Running the example` section below.
+
+When the application starts, a registered user, the movie watcher, is allocated a random preferred movie genre .
+
+The setup is defined in the [Setup.java](./src/main/java/io/quarkiverse/langchain4j/samples/chatbot/Setup.java) class.
+
+The registered movie watchers stored in a PostgreSQL database. When running the demo in dev mode (recommended), the database is automatically created and populated.
+
+The genre preferred by the registered movie watcher is used by a Movie `ContentRetriever` to sort the list of movies.
+
+## Running the example
+
+Users must authenticate with Google before they can start working with the Movie Muse and register the OpenAI API key.
```

Comment:
Please add some examples of suitable prompts that the users can try out
(for example, `Give me movies with a rating higher than 8` seems fine to me)